docs/reference/enrichement.mdx

```diff
@@ -0,0 +1,652 @@
+availability of the API server might be occasionally compromised, or you simply
+don't want to rely on it. The downside is that not all Kubernetes information is
+available through the container runtimes. Check the section [Communication with
+the Kubernetes API server](#communication-with-the-kubernetes-api-server) to
+ensure `ig` provides all the Kubernetes information you might need.
+
+Now, by default, `ig` doesn't show the Kubernetes information when using the
+`columns` output mode. It is because `ig` is mainly designed to be not
+Kubernetes specific. In the [Displaying the Kubernetes
+information](#displaying-the-kubernetes-information) section, you can see how to
+make the Kubernetes information visible with `ig`.
+
+</TabItem>
+</Tabs>
+
+### Displaying the Kubernetes information
+
+Inspektor Gadget doesn't show all the Kubernetes information by default when
+using the `columns` output mode. However, you can use the `columns` output mode
+along with the `fields` flag to display the Kubernetes-specific fields you are
+interested in. Or, you can use `--fields=+k8s` to display the default Gadget
+columns plus all the Kubernetes-specific fields. See the [Selecting specific
+fields](/docs/reference/run.mdx#selecting-specific-fields) section for more
+information.
+
+Alternatively, you can always get the full Kubernetes information by using any
+of the JSON or YAML [output formats](/docs/reference/run.mdx#output-format)
+Inspektor Gadget provides. In such cases, the Kubernetes information will be
+shown under the `k8s` group:
+
+<Tabs groupId="env">
+<TabItem value="kubectl-gadget" label="kubectl gadget">
+
+```bash
+$ kubectl gadget run trace_exec -o jsonpretty
+{
+  "args": "/bin/gadgettracermanager -liveness",
+  "comm": "gadgettracerman",
+  "cwd": "",
+  "error": "",
+  "error_raw": 0,
+  "gid": 0,
+  "group": "root",
+  "k8s": {
+    "containerName": "gadget",
+    "hostnetwork": false,
+    "namespace": "gadget",
+    "node": "aks-agentpool-19875870-vmss000001",
+    "owner": {
+      "kind": "DaemonSet",
+      "name": "gadget"
+    },
+    "podName": "gadget-9sjkj"
+  },
+  "loginuid": 4294967295,
+  "loginuser": "uid:4294967295",
+  "mntns_id": 4026532271,
+  "pcomm": "containerd-shim",
+  "pid": 483712,
+  "ppid": 483689,
+  "pupper_layer": false,
+  "runtime": {
+    "containerId": "491cc81598d021253edf154d0bf5d6b7d681f7810d326bde6b9c210506ba793f",
+    "containerImageDigest": "sha256:bcd2a52069e7816acba82c0fdf9b849f8f99bbfe37744a78a3e20240c02d65c1",
+    "containerImageName": "ghcr.io/blanquicet/gadget:latest",
+    "containerName": "gadget",
+    "runtimeName": "containerd"
+  },
+  "sessionid": 4294967295,
+  "tid": 483712,
+  "timestamp": "2024-08-30T12:12:25.621903770Z",
+  "timestamp_raw": 1725019945621903770,
+  "uid": 0,
+  "upper_layer": false,
+  "user": "root"
+}
+```
+
+</TabItem>
+<TabItem value="ig" label="ig">
+
+```bash
+$ ig run trace_exec -o jsonpretty
+{
```

Comment:
Remove things not related to kubernetes field to make it easier to understand? (same in other places?)